pywbem/_recorder.py

```diff
@@ -447,16 +622,17 @@ def __init__(self, fp):
         """
         Parameters:
 
-        fp (file):
-          An open file that each test case will be written to.  This file
-          should have been opened in text mode.
+          fp (file):
+            An open file that each test case will be written to.  This file
+            should have been opened in text mode.
+
+            Since there are differences between python 2 and 3 in opening
+            files in text mode, the static method
+            open_file(filename) can be used to open the file.
 
-          Since there are differences between python 2 and 3 in opening
-          files in text mode, the static method
-          open__file(filename) can be used to open the file.
+        Example::
 
-        Example:
-          recorder = TestClientRecorder(open_file('recorder.log')
+          recorder = TestClientRecorder(open_file('recorder.log'))
```

Comment:
The example does not work, because `open_file()` as a static method still needs either an object or the class to be invoked on. Change `open_file(...)` to `TestClientRecorder.open_file(...)`.